docs/specifications/tests/DNSSEC-TP/dnssec01.md

```diff
@@ -132,3 +141,5 @@ None.
 [DS_ALGORITHM_RESERVED]:   #outcomes
 [DS_ALGORITHM_OK]:         #outcomes
 [DS_ALGORITHM_MISSING]:    #outcomes
+[DS_ALGO_SHA1_DEPRECATED]:     #outcomes
+[Wikipedia]:                   https://en.wikipedia.org/wiki/SHA-1
```

Comment:
Maybe change the link label to include "SHA-1"? If you want the link text to be something other than the link label that can be achieved using a [full reference link](https://spec.commonmark.org/0.29/#full-reference-link).

Reply:
If there were multiple references to different locations at Wikipedia I would agree (and done that way) but in this case there is only one reference to Wikipedia so I do not see the gain.